components/Header.tsx

```diff
@@ -0,0 +1,75 @@
+import { SearchIcon } from './Icons';
+const Header = () => {
```

Comment:
Given that the only difference between the two is the word clubs/events, and where search queries will go, I'm wondering if it would simplify things to have a shared Header component.

Reply:
I kept it separate so that whenever the TRPC functionality gets implemented, then we can separately query the two

Reply:
Makes sense. We could reinvestigate this if needed once TRPC is implemented.